products/healthcare/terraform.yaml

```diff
@@ -0,0 +1,103 @@
+# Copyright 2019 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  Dataset: !ruby/object:Overrides::Terraform::ResourceOverride
+    description: |
+      {{description}}
+    id_format: "projects/{{project}}/locations/{{location}}/datasets/{{name}}"
+    import_format: ["projects/{{project}}/locations/{{location}}/datasets/{{name}}"]
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "healthcare_dataset_basic"
+        primary_resource_id: "default"
+        vars:
+          dataset_name: "example-dataset"
+          location: "us-central1"
+          time_zone: "America/New_York"
+    properties:
+      timeZone: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
+      creationTime: !ruby/object:Overrides::Terraform::PropertyOverride
+        exclude: true
+      selfLink: !ruby/object:Overrides::Terraform::PropertyOverride
+        ignore_read: true
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
+      decoder: templates/terraform/decoders/long_name_to_self_link.go.erb
+  FhirStore: !ruby/object:Overrides::Terraform::ResourceOverride
+    description: |
+      {{description}}
+    id_format: "{{dataset}}/fhirStores/{{name}}"
+    import_format: ["{{dataset}}/fhirStores/{{name}}"]
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "healthcare_fhir_store_basic"
+        primary_resource_id: "default"
+        vars:
+          dataset_name: "example-dataset"
+          fhir_store_name: "example-fhir-store"
+          pubsub_topic: "fhir-notifications"
+    properties:
+      creationTime: !ruby/object:Overrides::Terraform::PropertyOverride
+        exclude: true
+      selfLink: !ruby/object:Overrides::Terraform::PropertyOverride
+        ignore_read: true
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
+      decoder: templates/terraform/decoders/long_name_to_self_link.go.erb
+  DicomStore: !ruby/object:Overrides::Terraform::ResourceOverride
+    description: |
+      {{description}}
+    id_format: "{{dataset}}/dicomStores/{{name}}"
+    import_format: ["{{dataset}}/dicomStores/{{name}}"]
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "healthcare_dicom_store_basic"
```

Comment:
```suggestion
        name: "healthcare_dicom_store_basic"
        skip_test: true
```

Reply:
Why should I be skipping tests for this? Is it because you're suggesting I wrote update tests that would probably cover this?

Reply:
Ah - I should have explained better. If you exclude import the tests will fail to generate. So the two must be done in tandem. Until I fix the test generation so it will create valid tests :P

Reply:
Well I don't have ass anything so I wasn't about to enable "exclude_import" :-) . I've written the custom importers for the 3 stores.  I'm just working on the integration tests now (admittedly using the generated ones as the base to test changes was helpful).